spec/vttutils.spec.ts

```diff
@@ -349,7 +349,7 @@ describe('Vtt Utils', () => {
 
           expect(spyCss).toHaveBeenCalledTimes(11);
           expect(spyCss).toHaveBeenNthCalledWith(6, 'writing-mode', 'vertical-rl');
-          expect(spyCss).toHaveBeenNthCalledWith(7, 'left', '0');
+          expect(spyCss).toHaveBeenNthCalledWith(8, 'right', '0');
```

Comment:
here there is a new css mutation between the "writing-mode" and position, I didn't examine this